sessionctx/variable/tidb_vars.go

```diff
@@ -323,6 +323,11 @@ const (
 	// Note if you want to set `tidb_enforce_mpp` to `true`, you must set `tidb_allow_mpp` to `true` first.
 	TiDBEnforceMPPExecution = "tidb_enforce_mpp"
 
+	// TiFlashMaxThreads TiflashMaxThreads is the maximum number of threads to execute the request which is pushed down to tiflash.
+	// Default value is -1, means it will not be pushed down to tiflash.
+	// If the value is bigger than -1, it will be pushed down to tiflash and used to create db context in tiflash.
+	TiFlashMaxThreads = "max_threads"
```

Comment:
Sorry, one more change. For this we need the tidb prefix. Even for other TiFlash vars they currently start with TiDB something:

```suggestion
	TiFlashMaxThreads = "tidb_tiflash_max_threads"
```

Reply:
Feel free to chose another name if you can think of better of course. Using `tidb_tiflash` as a prefix seems kind of weird, but we are usually consistent with the `tidb_` prefix.

Reply:
> Feel free to chose another name if you can think of better of course. Using `tidb_tiflash` as a prefix seems kind of weird, but we are usually consistent with the `tidb_` prefix.

how about tidb_max_tiflash_threads?

Reply:
renamed.

Reply:
I agree, much better.